### PR TITLE
Rollback on ErrImageNeverPull, ErrImagePullBackOff and CrashLoopBackOff

### PR DESCRIPTION
**Pull request by lftakakura**

### CODE REVIEW

File: metadata/version.go [hunk 1]
```diff
@@ -8,7 +8,7 @@
 package metadata
 
 //Version of Maestro
-var Version = "8.1.1"
+var Version = "8.2.1"
```
Comment (andrehp) [inline metadata/version.go]: Why not 8.2.0?

File: models/constants.go [hunk 2]
```diff
@@ -102,3 +102,10 @@ const Global = "global"
 
 // PodNotFitsHostPorts is a message when the pod's host port is no available in any node of the pool
 const PodNotFitsHostPorts = "PodFitsHostPorts"
+
+// InvalidPodWaitingStates are all the states that are not accepted in a waiting pod.
+var InvalidPodWaitingStates = []string{
+	"ErrImageNeverPull",
+	"ErrImagePullBackOff",
+	"CrashLoopBackOff",
```
Comment (andrehp) [inline models/constants.go]: It might be useful to watch for `ErrInvalidImageName` as well

Comment (lftakakura) [reply models/constants.go]: ok! thanks

File: metadata/version.go (hunk 1 above)
Comment (lftakakura) [reply metadata/version.go]: 8.2.0 was released already. Forgot to change the metadata =P

**Pull request merged**